processing/src/main/java/org/apache/druid/query/aggregation/any/NumericNilVectorAggregator.java

```diff
@@ -69,6 +69,11 @@ public static NumericNilVectorAggregator longNilVectorAggregator()
   @Nullable
   private final Object returnValue;
 
+  public static NumericNilVectorAggregator of(Object returnValue)
```

Comment:
side note: I wonder if we should rename `NumericNilVectorAggregator` if we are going to be using it for non-numbers, maybe just `NilVectorAggregator`. I guess this doesn't need to be done in this PR, but it does seem kind of funny exposing it for other purposes